Block/RecentPostsBlockService.php

```diff
@@ -34,6 +34,11 @@ class RecentPostsBlockService extends BaseBlockService
     protected $manager;
 
     /**
+     * @var Pool
+     */
+    private $adminPool;
```

Comment:
If someone is relying on:

```
$service = new RecentPostsBlockService() //let's assume that it has no constructor
$pool = $service->adminPool;
```

With the old code that would work, with this change it will break.

No declared property => public property.